examples/MyModule/recording_backend_soundclick.h

```diff
@@ -0,0 +1,3082 @@
+/*
+ *  recording_backend_soundclick.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef RECORDING_BACKEND_SOUNDCLICK_H
+#define RECORDING_BACKEND_SOUNDCLICK_H
+
+// recording backend interface
+#include "recording_backend.h"
+
+// Simple and Fast Multimedia Library (SFML)
+#include <SFML/Audio.hpp>
+
+namespace nest
+{
+
+/**
+ * SoundClick specialization of the RecordingBackend interface.
+ *
+ * Recorded spike events produce a clicking sound similar to that heard in
+ * electrophysiological recordings. The backend works on a
+ * per-device-per-thread basis.
+ *
+ * RecordingBackendSoundClick makes use of the SFML
+ * (Simple and Fast Multimedia Library) audio module
+ * This requires lifsml-dev to be installed,
+ * e.g., sudo apt-get install libsfml-dev.
+ * (https://www.sfml-dev.org/)
+ *
+ * NOTE: To creates the illusion of a realistic sound from an
+ *       electrophysiological recording, this backend slows down
+ *       the simulation to biological real time!
+ */
+class RecordingBackendSoundClick : public RecordingBackend
+{
+public:
+  RecordingBackendSoundClick();
+
+  ~RecordingBackendSoundClick() throw();
+
+  // Register a device to record data using a certain backend
+  void enroll( const RecordingDevice& device );
+  void enroll( const RecordingDevice& device,
+    const std::vector< Name >& value_names );
+
+  // Clean up the backend at the end of a call to Simulate
+  void finalize();
+
+  // Synchronize backends at the end of each simulation cycle
+  void synchronize();
+
+  // Write the data from the event to the backend specific channel
+  void write( const RecordingDevice& device, const Event& event );
+  void write( const RecordingDevice& device,
+    const Event& event,
+    const std::vector< double >& );
+
+  // Initialize global backend-specific data structures
+  void initialize();
+
+private:
+  // NEST Stopwatch object
+  Stopwatch stopwatch_;
```

Comment:
I can't see the `include` for the `Stopwatch`. How does this work? Is this currently coming from one of the "unused" headers in the `.cpp` file?

Reply:
Made the include explicit.